src/libraries/Native/Unix/Common/pal_compiler.h

```diff
@@ -19,4 +19,8 @@
 #define c_static_assert(e) c_static_assert_msg(e, "")
 #endif
 
-#define DLLEXPORT __attribute__ ((__visibility__ ("default")))
+#ifdef __LIB_NATIVE_ENTRYPOINTS
+#define PALEXPORT __attribute__ ((__visibility__ ("hidden")))
```

Comment:
@jkotas did I understand correctly what you meant here? Or do this needs to always be "hidden" for System.Globalization.Native now that it is linked into coreclr, even when building the implementation?

Reply:
I think I got what you said, I just changed it to define it as "default" here if not defined and then in GlobalizationNative defined it to "hidden" before importing `pal_compiler.h`